src/common/TransactionStatusCell.css

```diff
@@ -10,3 +10,7 @@
     margin-left: 10px;
     padding-bottom: 0;
 }
+
+.TransactionStatusCellDetails > .Row {
+    margin-bottom: 0 !important;
```

Comment:
`!important` is a very bad practice. It is better to try to find a rule which is more specific.

Reply:
I know, but I have not been able to figure out which rule could do the same job...